ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/CompletedExceptionRequestModal.tsx

```diff
@@ -45,7 +45,7 @@ function expiryDisplay(
         case 'TIME': {
             if (expiry.expiresOn) {
                 // Since the expiry here will always be in the future, we don't need to check which date is earlier
-                const daysUntilExpiration = differenceInDays(expiry.expiresOn, new Date());
+                const daysUntilExpiration = differenceInCalendarDays(expiry.expiresOn, new Date());
```

Comment:
`differenceInDays` returns the number of full 24 hour periods between the two dates, which always results in one less day than we want for the display here.